Rakefile

```diff
@@ -1,5 +1,6 @@
 # Add your own tasks in files placed in lib/tasks ending in .rake,
-# for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
```

Comment:
Exactly. I only did it in that part. After that i disabled that verification:

Metrics/LineLength:
  Enabled: false

Temporally, until we come to an agreement in that issue.